src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.VsLanguageDebugInfo.cs

```diff
@@ -92,21 +92,25 @@ public int GetNameOfLocation(IVsTextBuffer pBuffer, int iLine, int iCol, out str
                             var textBuffer = _languageService.EditorAdaptersFactoryService.GetDataBuffer(pBuffer);
                             if (textBuffer != null)
                             {
-                                var point = textBuffer.CurrentSnapshot.GetPoint(iLine, iCol);
-                                var document = point.Snapshot.GetOpenDocumentInCurrentContextWithChanges();
-
-                                if (document != null)
+                                var nullablePoint = textBuffer.CurrentSnapshot.TryGetPoint(iLine, iCol);
```

Comment:
>nullablePoint [](start = 36, length = 13)

nit: i'd call this `pointOpt`, but this is fine too.

Reply:
Thanks! I tried to name it consistently with similar TryGetPosition and TryGetSpan from the same class returning nullables.

---
In reply to: [181254042](https://github.com/dotnet/roslyn/pull/26128#discussion_r181254042) [](ancestors = 181254042)